embedders/__init__.py

```diff
@@ -37,18 +40,30 @@ def transform(
         """
         pass
 
-
-class Embedder(Transformer):
     @abstractmethod
-    def __init__(self):
+    def get_warnings(self) -> List:
+        """Collects all warnings reported during the embedding creation or PCA.
+
+        Returns:
+            List: List with all warnings
+        """
         pass
```

Comment:
why is this an abstract method? 
I'd assume (though I might be wrong) that the warning is something always present since the init method initializes a warning field as array. and the get_warning method should always return the same thing?

Reply:
The get_warning method differs between Embedder and PCAReducer classes which both inherit from the Transformer class. In Embedder the warnings list is returned, while in PCAReducer warnings of the PCAReduces and the warnings of the contained Embedder object are returned.
For this reason, I decided to make the method abstract and implement the different versions in Embedder and PCAReducer.

Reply:
makes sense